api.planx.uk/helpers.ts

```diff
@@ -63,7 +65,7 @@ interface InsertFlow {
 }
 
 // Insert a new flow into the `flows` table
-const insertFlow = async (
+const createFlow = async (
```

Comment:
In my opinion, the editor store function `createFlow` _should_ be calling this API method instead of doing a frontend operation ! Have made a note of this for now, but intentionally choosing not to refactor here to keep momentum on templates as a whole - will revisit when adding methods for "createTemplatedFlow"

Reply:
Yep - agreed - we want to be narrowing this down to (ideally) a single implementation.